R/GraphLearner.R

```diff
@@ -102,6 +94,37 @@ GraphLearner = R6Class("GraphLearner", inherit = Learner,
         return(value$clone(deep = TRUE))
       }
       value
+    },
+    get_predict_type = function() {
```

Comment:
comment smth like "trace every path backward from the sink until a `x$predict_type` is found for each of these paths (or there are no predecessors)"